submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+%}
+
+Alright, so I tried 4 productivity 3 modules in an assembling machine 3 but this slowed down the machine so much that the processing energy was almost more than the fuel energy it put out! So next I tried to remove a productivity module and dampen some of the inflated power consumption with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed strategy and used a speed module to restore the speed of the machine and cut down the processing time. While this performed better than the efficiency module, it still did not make enough of a difference. I tried more combinations after that and even tried adding various beacon combinations too (no diagrams for those) but no matter what I only managed to get close to a breakeven. 
+
+Based on these examples, it seems that while modules do offer some significant energy gain boosts, they do not necessarily make every single fuel processing step better (it depends on the recipe crafting time vs. the energy gains).
+
+## So how useful is fuel processing in general?
+
+Our findings so far about energy gains only partially help us answer this question. I have identified a more comprehensive list of factors that you should consider when deciding whether to (further) process your burner fuels:
+
+* **Gains in energy:** If you get more energy out of the processed fuel, then your demand for the unprocessed fuel decreases and so your resource patches run out more slowly. We determined the energy gains for different fuels earlier.
+- **Gains in vehicle acceleration bonus and top speed bonus:** Greater bonuses mean that your vehicles can go faster. More processed fuels offer greater bonuses. 
+- **Gains in energy density (per unit and per stack):**  More energy per stack means the same inventory slot can hold more energy, which makes a significant difference for how much energy you can fit into each cargo wagon or chest. Meanwhile, more energy per unit means more energy can be carried by robots, belts, and inserters, which can only hold a few units at a time. If your energy consumption is fixed, using a more energy-dense fuel means needing less logistics infrastructure to move fuels, which means simpler designs and less energy consumed by the logistic system itself. More processed fuels have greater energy density in both regards, hence having these benefits.
+- **Cost of processing setup:** This is in terms of materials and space needed to make enough buildings and logistics for the processing, as well as the unlock cost of the required technology. Each tier of processing adds to these costs.
+- **Availability of more coal/oil to just use instead:** Since these resources are technically infinite on most maps, you may prefer to skip the energy gains and brute force your fuel shortages by setting up more resource extraction infrastructure instead of more fuel processing infrastructure. This factor simply depends on your surroundings.
```

Comment:
```suggestion
- **Availability of more coal/oil to just use instead:** Since these resources are technically infinite on most maps, you may prefer to skip the energy gains and brute force your fuel shortages by setting up more resource extraction infrastructure, instead of building more fuel processing infrastructure. This factor simply depends on your surroundings.
```